msbuild/tests/Xamarin.iOS.Tasks.Tests/TaskTests/MTouchTaskTests.cs

```diff
@@ -52,6 +63,7 @@ public override void Setup ()
 			Task.IntermediateOutputPath = Path.Combine ("obj", "mtouch-cache");
 			Task.MainAssembly = new TaskItem ("Main.exe");
 			Task.References = new [] { new TaskItem ("a.dll"), new TaskItem ("b.dll"), new TaskItem ("c.dll") };
```

Comment:
Maybe add a `new TaskTiem ("with spaces.dll")` too to make sure the response file contains quotes when needed.

Reply:
Might also be worth adding a task item that has quotes in the name to make sure that gets quoted (and escaped) correctly as well.